### PR TITLE
Chore/0000001 update readme

### PR DESCRIPTION
**Pull request by theinfamousrj**
Added a good bit to the README so that we all know what's going on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
 # sygaldry
+
+## Building Sygaldry
+```bash
+go build -o sygaldry
+```
+
+## Dependencies
+```bash
+go get -u -v github.com/smallfish/simpleyaml
```
Comment (bricerising) [inline README.md:10]: I think we should use "deps" for dependencies so we can have a file that docs and keeps up with our dependencies

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+
+## Using the CLI
+```bash
+sygaldry build -f mySygaldryYamlFile.yaml
+```
+
+## YAML
+
+### Formatting
+```yaml
```
Comment (bricerising) [inline README.md:21]: We may want to wait until we have some real runes before we document these

Comment (theinfamousrj) [reply README.md:21]: We can do that. I just figured first pass this was stuff we had talked about.

File: README.md (hunk 1 above)
Comment (theinfamousrj) [reply README.md:10]: [Dis wun](https://www.freecodecamp.org/news/an-intro-to-dep-how-to-manage-your-golang-project-dependencies-7b07d84e7ba5/)?

Comment (dizveloper) [reply README.md:10]: use go mods instead (:

**Review by bricerising — APPROVED** [review]

**Pull request merged**